assets/src/application/ui/display/TimezoneTimeInfo/style.scss

```diff
@@ -1,6 +1,15 @@
 .ee-timezone-info {
 	display: block;
-	margin: 0 var(--ee-margin-tiny);
+	margin: 0 auto;
+	position: relative;
+	[role='dialog'] {
+		position: absolute;
+		min-width: 260px;
+		left: 0;
+		[x-arrow] {
+			display: none;
```

Comment:
the reasons for this: 
![image](https://user-images.githubusercontent.com/1477453/79157547-70af4300-7ddd-11ea-8efb-2bcbbecb3921.png)

Reply:
I tried manually to  hack this, but when when start and end dates are displayed it has a different position: 

![image](https://user-images.githubusercontent.com/1477453/79157642-a05e4b00-7ddd-11ea-90d8-9a97d30f72e4.png)

Reply:
was thinking that maybe it's more stable to use popover from base UI:

https://baseweb.design/components/popover/

Reply:
that's weird... wonder what was conflicting...